cache/blobs/blobs.go

```diff
@@ -72,15 +76,31 @@ func GetDiffPairs(ctx context.Context, snapshotter snapshot.Snapshotter, differ
 				return nil, err
 			}
 			descr, err := differ.DiffMounts(ctx, lower, upper,
-				diff.WithMediaType(ocispec.MediaTypeImageLayer),
-				diff.WithReference(ref.ID()))
+				diff.WithMediaType(ocispec.MediaTypeImageLayerGzip),
+				diff.WithReference(ref.ID()),
+				diff.WithLabels(map[string]string{
+					"containerd.io/gc.root": time.Now().UTC().Format(time.RFC3339),
```

Comment:
as stevvooe suggested in #155, this should be RFC3339Nano (until switching to lease API)